dulwich/porcelain.py

```diff
@@ -354,11 +435,14 @@ def commit(
         author = author.encode(encoding or DEFAULT_ENCODING)
     if getattr(committer, "encode", None):
         committer = committer.encode(encoding or DEFAULT_ENCODING)
+    author_timezone, commit_timezone = get_user_timezone()
```

Comment:
this should probably also allow specifying the two timezones as argument

Reply:
Resolved in 4ca95f5d56ff50b31c3c465cac1c256984a66b08